tidb-configuration-file.md

```diff
@@ -270,19 +270,19 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 
 ### `max-memory`
 
-+ Prepare cache LRU 使用的最大内存限制，超过 performance.max-memory * (1 - prepared-plan-cache.memory-guard-ratio) 会剔除 LRU 中的元素。
++ Prepare plan cache LRU 使用的最大内存限制，超过 performance.max-memory * (1 - prepared-plan-cache.memory-guard-ratio) 会剔除 LRU 中的元素。
```

Comment:
So the comment for this option in the config template is wrong.

Reply:
https://github.com/pingcap/tidb/blob/master/config/config.toml.example#L200

Reply:
Seems this variable is used by them both, https://github.com/pingcap/tidb/blob/master/tidb-server/main.go#L593.
Maybe we should change the variable name to distinguish them?

Reply:
We have prepared plan cache capacity already.
Maybe we don't need this value for prepared plan cache.

Reply:
Related [tidb-issue](https://github.com/pingcap/tidb/issues/17298)

Reply:
> Fix the annotation in tidb.toml.example and tell the readers that max-memory is only for Plan Cache in tidb.toml.example in all release branches(v2.1, v3.0, v3.1, v4.0).

The [issue](https://github.com/pingcap/tidb/issues/17298#issuecomment-632180632) said that `max-memory` is only for Plan Cache, we should change this documentation back.

Reply:
Fixed in https://github.com/pingcap/docs-cn/pull/3421